rust-src/aggregate_sig/benches/bench.rs

```diff
@@ -89,6 +89,38 @@ fn bench_verify_aggregate_sig(c: &mut Criterion) {
     });
 }
 
+fn bench_verify_aggregate_sig_hybrid(c: &mut Criterion) {
+    let mut csprng = thread_rng();
+    // number of signers for each message.
+    let num_signers = vec![200, 200];
```

Comment:
why is 200 reasonable? That's one data point.

A better benchmark would be to use different n's, e.g, `10, 20, 40, 100, 200, 400`. That's straightforward to set up.